src/EVM.hs

```diff
@@ -356,7 +357,7 @@ exec1 = do
               -- Move back the gas to the parent context
               assign (state . gas) (the state gas)
 
-            CallContext _ _ _ _ _ -> do
+            CallContext _ _ _ _ _ _ -> do
```

Comment:
I'll change these to use a syntax that doesn't need more and more underscores. :)